internal/statetransition/state_transition.go

```diff
@@ -627,7 +627,7 @@ func CalculateWorkReportsAndAccumulate(header *block.Header, currentState *state
 		//      ⎧ a except a′_a = τ′ if s ∈ K(S)
 		// a′ = ⎨
 		//      ⎩ a otherwise
-		if _, ok := accumulationStats[serviceId]; !ok {
+		if _, ok := accumulationStats[serviceId]; ok {
```

Comment:
@danielvladco I think you need to double check this fix. :) Traces were failing because this was incorrectly setting the most recently accumulated timeslot before. In the traces this just stays 0.

Reply:
it's perfect, I also made this fix on my branch 👍